experimental/packages/otlp-proto-exporter-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/otlp-proto-exporter-base",
-  "version": "0.27.0",
+  "version": "0.28.0",
   "description": "OpenTelemetry OTLP-HTTP-protobuf Exporter base (for internal use only)",
   "main": "build/src/index.js",
```

Comment:
some modules have additionally entries for `module` and `esnext` here.
Are there needed also here?

Reply:
I think not. They don't exist in the proto/grpc trace/metrics exporters before. I think because they aren't supported in browser environments.